lib/string.gi

```diff
@@ -1258,9 +1258,86 @@ InstallGlobalFunction(StringOfMemoryAmount, function(m)
     Append(s, units[shift+1]);
     return s;
 end);
-
-        
     
+InstallGlobalFunction(FormatTo, function(stream, s, args)
+    local pos, len, outstr, nextbrace, endbrace, id, argcounter;
```

Comment:
Is there a (good) reason why you don't use `args...`?

There is the option to pass a record, but that can be handled even in the presence of `args...`

Reply:
Good question. I started off with a list, but there isn't really a good reason for that. I'll switch to args...